src/rhtap/core/integration/git/providers/github.ts

```
@@ -1,19 +1,19 @@
 import { PullRequest } from '../models';
 import { ITemplate, TemplateFactory, TemplateType } from '../templates/templateFactory';
 import sodium from 'sodium-native';
-import { ContentModifications } from '../../../../modification/contentModification';
+import { ContentModifications } from '../../../../../common/modification/contentModification';
 
 export class GithubProvider extends BaseGitProvider {
   private githubClient!: GithubClient;
   private template!: ITemplate;
   private repoOwner: string;
-  private clientFactory: GitHubClientFactory;
+  private githubClient: GithubClient;
```

Biome (2.1.2)

[error] Duplicate class member name "githubClient"



(lint/suspicious/noDuplicateClassMembers)